### DISCUSSION TITLE
What parameters will affect the curve of PK peak?

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1364] [Questions & Problems]
What parameters will affect the curve of PK peak? Why is the peak in my figure like this?And I want to ask whether there are other parameters related to the change of PK curve?

Thank you in advance :)

![捕获](https://user-images.githubusercontent.com/109507330/197935295-a2bf3081-f1b8-4017-99e0-1e0c11dd6ccf.JPG)

**Comment by tobiasK2001** [comment]
Hi,

well there are quite a lot parameters possible: Type of administration, Dose, Reference Solubility, Lipophilicity, Clearance, Gastro intestinal transit time .... basically all the things mentioned here:
https://docs.open-systems-pharmacology.org/mechanistic-modeling-of-pharmacokinetics-and-dynamics/modeling-concepts/principles-of-pbpk-modeling
So we would need  bit more info from your side to be able to help more concretely. Ideally you upload your model file.

Best, Tobias
